node/src/accountant/mod.rs

```diff
@@ -1532,151 +1502,127 @@ pub mod tests {
-                .accountant(accountant_mock)
-                .build();
-            let subject = make_subject(
-                Some(bc_from_ac_plus_earning_wallet(
-                    AccountantConfig {
-                        payable_scan_interval: Duration::from_millis(100),
-                        payment_received_scan_interval: Duration::from_secs(10_000),
-                    },
-                    earning_wallet.clone(),
-                )),
-                Some(payable_dao),
-                None,
-                None,
-                None,
-            );
-            let subject_addr = subject.start();
-            let subject_subs = Accountant::make_subs_from(&subject_addr);
-
-            send_bind_message!(subject_subs, peer_actors);
-            send_start_message!(subject_subs);
-
-            system.run();
-        });
+            },
+            PayableAccount {
+                wallet: make_wallet("foo"),
+                balance: PAYMENT_CURVES.balance_to_decrease_from_gwub + 100,
+                last_paid_timestamp: from_time_t(PAYMENT_CURVES.payment_suggested_after_sec + 5),
+                pending_payment_transaction: None,
+            },
+        ];
+        let payable_dao = PayableDaoMock::new().non_pending_payables_result(accounts.clone());
+        let system = System::new(
+            "accountant_scans_for_received_payments_and_sends_message_to_blockchain_bridge",
+        );
+        let subject = make_subject(
+            Some(bc_from_ac_plus_earning_wallet(
+                AccountantConfig {
+                    payables_scan_interval: Duration::from_secs(100_000), //scan intervals deliberately big to demonstrate that we don't do the intervals yet
+                    receivables_scan_interval: Duration::from_secs(100_000),
+                },
+                earning_wallet.clone(),
+            )),
+            Some(payable_dao),
+            Some(receivable_dao),
+            None,
+            None,
+        );
+        let accountant_addr = subject.start();
+        let accountant_subs = Accountant::make_subs_from(&accountant_addr);
+        let peer_actors = peer_actors_builder()
+            .blockchain_bridge(blockchain_bridge)
+            .build();
 
-        TestLogHandler::new()
-            .await_log_containing("WARN: Accountant: Failed to send transaction", 1000u64);
+        send_bind_message!(accountant_subs, peer_actors);
+        send_start_message!(accountant_subs);
 
-        let accountant_recording = accountant_recording_arc.lock().unwrap();
-        assert_eq!(0, accountant_recording.len());
+        System::current().stop();
+        system.run();
+        let blockchain_bridge_recorder = blockchain_bridge_recording_arc.lock().unwrap();
+        assert_eq!(blockchain_bridge_recorder.len(), 2); //the second message is from the scan for receivables
+        let scans_for_received_payments_msgs: Vec<&RetrieveTransactions> = (0
```

Comment:
This variable (and the test result in one) should rather be named by retrieve_transactions_msgs;